client/app/assets/less/ant.less

```diff
@@ -8,6 +8,7 @@
 @import '~antd/lib/button/style/index.less';
 @import '~antd/lib/radio/style/index.less';
 @import '~antd/lib/time-picker/style/index.less';
+@import 'inc/variables';
```

Comment:
Don't we want to go "full ant style"? Is this meant to be dropped on react/ant migration completion?

Reply:
The idea is to adopt Ant components, but still retain our styling (or a better version of it).

Reply:
In that case, we should create an ant specific `variables` file excluding bootstrap-only vars, no?

Reply:
> In that case, we should create an ant specific variables file excluding bootstrap-only vars, no?

Makes sense to me, just now I figured that there is a `STYLING-README.md` that suggests to "Keep less/inc folder untouched, rewrite things in less/redash respectively". Following that, I think we currently don't need a separate or shared variable file for Ant and the rest (only a few are shared, we could use `ant.less` for this purpose).

I will push a commit with this and we can follow from there :grin:

Reply:
Aren't unused variables get dropped by the less compiler?

Reply:
Can use [this](https://gist.github.com/badsyntax/6193491#gistcomment-1323598) to find the unused btw.

Reply:
The problem is that if I import all of `inc/variables` the variables that exist in there would replace Ant Style with no control of what's changed. (i.e.: the font size you mentioned above, that was replaced by a variable in `inc/variables`)

Reply:
In this case, @ranbena is correct - we should have a separate variables file...

As for the STYLING_README file: I didn't realize we had one 😅 _But_ this comment about not touching `less/inc` feels wrong to me. This is _our_ stylesheet, while should be add overrides and not edit it?

Reply:
I thought about using `ant.less` mostly for the purpose of replacing Ant variables. However I agree that this is important and eventually would show up.

The organization in `ant.less` still annoys me a bit, but for the purpose of testing we have an almost pure Ant styling (a few changes centered in `ant.less`). We can now from here orchestrate things better :+1:

Reply:
@gabrieldutra Levko is going to have a separate PR for updating Antd (it's got some select feature he needs) so wait for that to happen in a bit.

There is one discrepancy @arikfr noticed - datepickers are 32px high while inputs are 35px high and when next to each other it's prominent. Perhaps the `@input-height-base: 35px;` override is not being applied.

Still, @arikfr and I both want this landed asap and the rest will be taken care of afterwards.
So once @kravets-levko's Antd upgrade PR lands, rebase, the tests will pass and you're good to go.

Reply:
Good, as soon as I have my laptop in hands I'll give it a work. The current version has none of the old variables, so this was expected, do you prefer having it with the import for all variables, or we should go adding them according to our need?